src/test/shell/integration/aquery_test.sh

```diff
@@ -1659,6 +1665,49 @@ EOF
   fi
 }
 
+function test_does_not_fail_horribly_with_file() {
```

Comment:
@kshyanashree This test made it in by mistake, only the one after it is part of the original PR.

Reply:
Hi @fmeum! I removed this test and ran the presubmits, but still checks are failing. Could you please look into it?